ts/nni_manager/training_service/reusable/trialDispatcher.ts

```diff
@@ -507,6 +507,7 @@ class TrialDispatcher implements TrainingService {
                                 throw new Error(`${environment.id} does not has environment service!`);
                             }
                             await environment.environmentService.stopEnvironment(environment);
+                            liveEnvironmentsCount--;
```

Comment:
I'm not really sure whether this is correct.
Suggest to merge first and I'll test it later.

Reply:
got it.